cooperator_worker_force/models/res_partner.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 Coop IT Easy SC
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+
+    _inherit = "res.partner"
+    worker_store = fields.Boolean(
```

Comment:
Shouldn't this field be name `worker_force` ?

Reply:
Okay, will do. I was feeling lazy ^^

Reply:
Actually, I'll leave it for [later](https://github.com/beescoop/Obeesdoo/issues/471): I'm not sure how migration scripts will behave since this module will be installed at production deployment.